lib/src/lib.rs

```diff
@@ -278,25 +244,24 @@ allow {
     }
 
     async fn attestation() -> Result<(), String> {
-        let service = Service::new().unwrap();
+        let service = Service::new();
 
         let evidence = sample_evidence();
-        let res = service.attestation(&evidence, None, None).await;
+        let res = service
+            .attestation(&evidence, None, Some(sample_reference(1)))
+            .await;
+        println!("{:?}", res.as_ref());
         assert!(res.is_ok(), "attestation should success");
         let v: Value = serde_json::from_str(&res.unwrap()).unwrap();
-        assert!(v["result"] == "true".to_string(), "result should true");
-        let policy_info: Value = serde_json::from_str(v["policy_info"].as_str().unwrap()).unwrap();
-        assert!(
-            policy_info["policy_name"] == "policy.rego".to_string(),
-            "policy name should be policy.rego"
-        );
+        assert!(v["result"] == json!(true), "result should true");
```

Comment:
if `assert!` prints the expression (upon error), then there's no need for the "reason" string (2nd argument), because it doesn't explain a lot more than the expression itself) (and same for line 264 below).